### PR TITLE
French translations for QA Choosing Encodings & HTML Encoding Declarations

### PR DESCRIPTION
**Pull request by clavoline**
Preview:

* https://deploy-preview-461--i18n-drafts.netlify.app/questions/qa-choosing-encodings.fr.html
* https://deploy-preview-461--i18n-drafts.netlify.app/questions/qa-html-encoding-declarations.fr.html

### CODE REVIEW

File: questions/qa-html-encoding-declarations.en.html [hunk 1]
```diff
@@ -190,16 +190,16 @@ <h3>Working with legacy HTML formats</h3>
...
     <p>HTML5 deprecated the use of the <code class="kw" translate="no">charset</code> attribute on an <code class="kw" translate="no">a</code> or <code class="kw" translate="no">link</code> element, so you should avoid using it. It originated in the HTML&nbsp;4.01 specification for use with the <code class="kw" translate="no">a</code>, <code class="kw" translate="no">link</code> and <code class="kw" translate="no">script</code> elements and was supposed to indicate the encoding of the document you are linking to. </p>
     <p>It was intended for use on an embedded link element like this:</p>
     <figure class="example">
-      <div class="badcode"><span><img src="../icons/dontcopy.png" alt=" "> Bad code. Don't copy!</span> </div>
-      <div><code>See our &lt;a href=&quot;/mysite/mydoc.html&quot; charset=&quot;iso-8859-15&quot;&gt;list of publications&lt;/a&gt;.</code></div>
+      <div class="badcode"><span><img src="../icons/dontcopy.png" alt="X"> Bad code. Don't copy!</span> </div>
+      <div><code class="language-html">See our &lt;a href=&quot;/mysite/mydoc.html&quot; charset=&quot;iso-8859-15&quot;&gt;list of publications&lt;/a&gt;.</code></div>
```
Comment (xfq) [inline questions/qa-html-encoding-declarations.en.html:202]: We should use `<pre><code>` according to https://www.w3.org/International/i18n-activity/guidelines/editing#syntax-highlighting

Comment (r12a) [reply questions/qa-html-encoding-declarations.en.html:202]: I wonder whether the style guidelines need changing.  Sometimes it's useful to be able to wrap the code as the window width shrinks, and this looks like one of those occasions.

**Comment by r12a** [comment]
@clavoline i would carry on as before for now.  We can easily change the markup later ourselves.

File: questions/qa-html-encoding-declarations.en.html (hunk 1 above)
Comment (xfq) [reply questions/qa-html-encoding-declarations.en.html:202]: I think that is the usage recommended by the HTML Standard, but I don't have a strong preference, and we can also use CSS (`white-space: pre-wrap;`) to control it.

Comment (r12a) [reply questions/qa-html-encoding-declarations.en.html:202]: I opened an issue at https://github.com/w3c/i18n-drafts/issues/464 where we can discuss this further without holding back @clavoline.

**Comment by r12a** [comment]
All now installed and announced.  Thanks @clavoline !

**Pull request merged**